ultraplot/axes/geo.py

```diff
@@ -480,6 +501,255 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
 
+    @override
+    def _sharey_limits(self, sharey: "GeoAxes"):
+        return self._share_limits(sharey, which="y")
+
+    @override
+    def _sharex_limits(self, sharex: "GeoAxes"):
+        return self._share_limits(sharex, which="x")
+
+    def _share_limits(self, other: "GeoAxes", which: str):
+        """
+        Safely share limits and tickers without resetting things.
+        """
+        # NOTE: See _sharex_limits for notes
+        if which == "x":
+            this_ax = self._lonaxis
+            other_ax = other._lonaxis
+        else:
+            this_ax = self._lataxis
+            other_ax = other._lataxis
+        for ax1, ax2 in ((this_ax, other_ax), (other_ax, this_ax)):
+            ax1.set_view_interval(*ax2.get_view_interval())
+
+        # Set the shared axis
+        getattr(self, f"share{which}")(other)
+
+        props = ["isDefault_majloc", "isDefault_minloc", "isDefault_majfmt"]
+        funcs = [
+            "major_locator",
+            "minor_locator",
+            "major_formatter",
+        ]
+        for prop, func in zip(props, funcs):
+            if getattr(other_ax, prop) and not getattr(this_ax, prop):
+                setter = getattr(other_ax, f"set_{func}")
+                getter = getattr(this_ax, f"get_{func}")
+                if setter and getter:
+                    setter(getter())
```

Comment:
Again missing coverage in these new functions points to incomplete tests or a bug.

Reply:
Fixed!